app/controllers/v0/caregivers_assistance_claims_controller.rb

```diff
@@ -5,6 +5,8 @@ module V0
   class CaregiversAssistanceClaimsController < ApplicationController
     skip_before_action(:authenticate)
 
+    rescue_from ::Form1010cg::Service::InvalidVeteranStatus, with: :backend_service_outage
```

Comment:
Wondering if you could extend the rescue_from behavior from the application_controller.rb to rescue_from Common::Exceptions::ServiceOutage instead of raising InvalidVeteranStatus https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/controllers/application_controller.rb#L115

Render errors is called here: https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/controllers/application_controller.rb#L126

On a side note: Does Backend Service Outage make sense here when an ICN isn't found?

Reply:
I think I answered my question as to why Backend Service Outage Exception is being raised from the attached ticket: " we need to ensure that the same error appears regardless of why the application was rejected " -- Makes sense.

Still curious on the first question though regarding the rescue_from behavior.

Reply:
Yes, the backend outage error is for ambiguity.

Regarding the **rescue_from** logic:

The `ApplicationController` seems out of scope for this. I could see it being moved to the `ApplicationController` if multiple controllers needed to handle this error. This is a special case, so I did not want to fill the application-wide error handling logic with a one-off exception.

I chose to use the custom error `Form1010cg::Service::InvalidVeteranStatus` for clarity. It's easier to interpret the source of the error. If I were to raise a `Common::Exceptions::ServiceOutage` instead, it could mislead a developer to thinking an external API call raised an error (if not looking at the stack trace). With this error, it's explicitly written in the controller that "InvalidVeteranStatus renders as a BackendServiceOutage."

Reply:
Ok, great! WRT clarity, I think this makes sense! LGTM